items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -121,6 +121,31 @@ class SierraItemUpdater(sierraSource: SierraSource)(
             case None            => item
           }
       }
-    } yield updatedItems.toSeq
+
+      updatedItemsWithAvailableDates = updatedItems.map {
+        case (item) =>
+          val displayAccessConditionOps = new DisplayAccessConditionOps(
+            item.locations.head.accessConditions.head
```

Comment:
Because this is an implicit class, so long as it's imported you don't need to construct it: `isRequestable` will just appear on any instance of `DisplayAccessCondition`! https://docs.scala-lang.org/overviews/core/implicit-classes.html#usage